tests/tests/bitasset_tests.cpp

```diff
@@ -542,6 +544,455 @@ BOOST_AUTO_TEST_CASE( hf_890_test )
+   BOOST_TEST_MESSAGE( "Prediction market with different precision" );
+   op.asset_to_update = prediction->get_id();
+   op.issuer = prediction->issuer;
+   BOOST_CHECK(evaluator.evaluate(op) == void_result());
+   op.asset_to_update = bit_usd_id;
+   op.issuer = bit_usd.issuer;
+
+   // checking old backing asset instead of new backing asset
+   BOOST_TEST_MESSAGE( "Was checking old backing asset instead of new backing asset" );
+   op.new_options.short_backing_asset = sixbit->get_id();
+   BOOST_CHECK(evaluator.evaluate(op) == void_result());
+   op.new_options.short_backing_asset = prediction->get_id();
+   BOOST_CHECK(evaluator.evaluate(op) == void_result());
+   op.new_options.short_backing_asset = correct_asset_id;
+
+   // Feed lifetime must exceed block interval
+   BOOST_TEST_MESSAGE( "Feed lifetime less than or equal to block interval" );
+   const auto good_feed_lifetime = op.new_options.feed_lifetime_sec;
+   op.new_options.feed_lifetime_sec = db.get_global_properties().parameters.block_interval;
+   BOOST_CHECK(evaluator.evaluate(op) == void_result());
+   op.new_options.feed_lifetime_sec = db.get_global_properties().parameters.block_interval - 1; // default interval > 1
+   BOOST_CHECK(evaluator.evaluate(op) == void_result());
+   op.new_options.feed_lifetime_sec = good_feed_lifetime;
+
+   // Force settlement delay must exceed block interval.
+   BOOST_TEST_MESSAGE( "Force settlement delay less than or equal to block interval" );
+   const auto good_force_settlement_delay_sec = op.new_options.force_settlement_delay_sec;
+   op.new_options.force_settlement_delay_sec = db.get_global_properties().parameters.block_interval;
+   BOOST_CHECK(evaluator.evaluate(op) == void_result());
+   op.new_options.force_settlement_delay_sec = db.get_global_properties().parameters.block_interval - 1; // default interval > 1
+   BOOST_CHECK(evaluator.evaluate(op) == void_result());
+   op.new_options.force_settlement_delay_sec = good_force_settlement_delay_sec;
+
+   // this should pass
+   BOOST_TEST_MESSAGE("We should be all good again.");
+   BOOST_CHECK( evaluator.evaluate(op) == void_result() );
+}
+
+/******
+ * @brief Test various bitasset asserts within the asset_evaluator before the HF 922 / 931
+ */
+BOOST_AUTO_TEST_CASE( bitasset_evaluator_test_after_922_931 )
+{
+   // set up the basics for the test
+   fc::log_level log_level_all(fc::log_level::all);
+
+   BOOST_TEST_MESSAGE("Advance to after hard fork 922 / 931");
+   auto maint_interval = db.get_global_properties().parameters.maintenance_interval;
+   generate_blocks( HARDFORK_CORE_922_931_TIME + maint_interval);
+   trx.set_expiration( HARDFORK_CORE_922_931_TIME + maint_interval + fc::hours(1));
+
+   ACTORS((nathan)(john));
+
+   BOOST_TEST_MESSAGE("Create USDBIT");
+   const asset_object& bit_usd = create_bitasset("USDBIT", GRAPHENE_COMMITTEE_ACCOUNT);
+   asset_id_type bit_usd_id = bit_usd.id;
+
+   BOOST_TEST_MESSAGE( "Create market issued BLAH" );
```

Comment:
BLAH is user-issued, not market-issued, see above.

Since you use the same initialization in both tests, perhaps move the asset creation into a common function.

Reply:
Created a function for asset creation